docs/interactions/Slash_Commands.md

```diff
@@ -0,0 +1,839 @@
+| value?   | OptionType                                       | the value of the pair                           |
+| options? | array of ApplicationCommandInteractionDataOption | present if this option is a group or subcommand |
+
+###### Interaction Response
+
+After receiving an interaction, you must respond to acknowledge it. This may be a `pong` for a `ping`, a message, or simply an acknowledgement that you have received it and will handle the command async.
+
+Interaction responses may choose to "eat" the user's command input if you do not wish to have their slash command show up as message in chat. This may be helpful for slash commands, or commands whose responses are asynchronous or ephemeral messages.
+
+| Field | Type                                      | Description                  |
+|-------|-------------------------------------------|------------------------------|
+| type  | InteractionResponseType                   | the type of response         |
+| data? | InteractionApplicationCommandCallbackData | an optional response message |
+
+###### InteractionResponseType
+
+| Name                     | Value | Description                                                       |
+|--------------------------|-------|-------------------------------------------------------------------|
+| Pong                     | 1     | ACK a `Ping`                                                      |
+| Acknowledge              | 2     | ACK a command without sending a message, eating the user's input  |
+| ChannelMessage           | 3     | respond with a message, eating the user's input                   |
+| ChannelMessageWithSource | 4     | respond with a message, showing the user's input                  |
+| ACKWithSource            | 5     | ACK a command without sending a message, showing the user's input |
+
+###### InteractionApplicationCommandCallbackData
+
+Not all message fields are currently supported.
+
+| Name              | Value            | Description                                                                                 |
+|-------------------|------------------|---------------------------------------------------------------------------------------------|
+| tts?              | bool             | is the response TTS                                                                         |
+| content           | string           | message content                                                                             |
+| embeds?           | array of embeds  | supports up to 10 embeds                                                                    |
+| allowed_mentions? | allowed mentions | [allowed mentions](#DOCS_RESOURCES_CHANNEL/allowed-mentions-object) object                  |
+| flags             | int              | acceptable values are [message flags](#DOCS_RESOURCES_CHANNEL/message-object-message-flags) |
```

Comment:
this can only be used to set the EPHEMERAL flag; probably shouldn't document it at all if ephemeral messages arent being documented